package.json

```diff
@@ -45,20 +42,24 @@
     "get-stdin": "^7.0.0",
     "google-gax": "^2.3.1",
     "js-yaml": "^3.13.1",
+    "module-alias": "^2.2.2",
```

Comment:
Adding a runtime dependency in pursuit of a build toolchain change doesn't feel right

Reply:
Agreed, like 100% agreed. This is all about the `protos.js` that get compiled and placed into random location, which TypeScript can figure out during compile time but Node cannot find in the runtime. Let me add a huge TODO here, we must get rid of this, but as of today, I don't have a good idea how exactly to do it.